test/api_compatibility.js

```diff
@@ -7,30 +7,14 @@ var expect = require('chai').expect;
 // library.  Again, globals are used purely for convenience.
 var opentracing = require('../debug.js');
 
-module.exports = function apiCompatibilityChecks(createTracer) {
-    describe('OpenTracing API', function() {
-        describe('Constants', function() {
-            var constStrings = [
-                'FORMAT_TEXT_MAP',
-                'FORMAT_BINARY',
-                'FORMAT_HTTP_HEADERS',
-                'REFERENCE_CHILD_OF',
-                'REFERENCE_FOLLOWS_FROM',
-            ];
-            _.each(constStrings, function(name) {
-                it(name + ' should be a constant string', function() {
-                    expect(opentracing[name]).to.be.a('string');
-                });
-            });
-        });
+module.exports = function apiCompatibilityChecks(_createTracer) {
+    var createTracer = _createTracer || function() { return opentracing.Tracer(); };
 
+    describe('OpenTracing API', function() {
         describe('Standalone functions', function() {
             var funcs = [
                 'childOf',
                 'followsFrom',
```

Comment:
should these really be on each individual implementation? I would've expected them to be always used from the opentracing namespace.